src/main/java/lotto/domain/Lotto.java

```diff
@@ -13,13 +13,21 @@
 
     private final List<LottoNumber> lottoNumbers;
 
+    private Lotto() {
+        this(LottoAutoGenerator.getInstance().generateLotto());
+    }
+
+    public static Lotto autoLotto() {
+        return new Lotto();
+    }
```

Comment:
아래와 같이 변경하는 것이 더 좋을 것 같아요. 🤔 

```suggestion
    public static Lotto autoLotto() {
        LottoAutoGenerator lottoAutoGenerator = LottoAutoGenerator.getInstance();
        return new Lotto(lottoAutoGenerator.generateLotto());
    }
```